src/hypofuzz/cov.py

```diff
@@ -50,6 +57,8 @@ def get_coverage_instance(**kwargs: Any) -> coverage.Coverage:
 
 is_hypothesis_file = belongs_to(hypothesis)
 is_hypofuzz_file = belongs_to(hypofuzz)
+is_pytest_file = belongs_to(pytest)
+is__pytest_file = belongs_to(_pytest)
```

Comment:
Hmm, we should check that `belongs_to()` memoizes so that the returned functions share a cache.

Reply:
each `belongs_to` has its own cache, not shared. Would sharing help? Knowing that an `fpath: str` does or doesn't belong to `pytest` doesn't tell us anything about whether it belongs to `_pytest`, right?

Reply:
I mean we have `is_hypothesis_file = belongs_to(hypothesis)` defined in both Hypothesis and HypoFuzz, and iirc those are cached separately.  Not a big deal either way, of course.

Reply:
Ah, right. Yeah, we probably should make `belongs_to` share a cache, and make it lru while we're at it.